notebooks/GWAS_1000_genomes_p1/main.py

```diff
@@ -0,0 +1,137 @@
+
+    ## Set up your notebook
+    ----
+
+    ### Set runtime values
+    If you are opening this notebook for the first time, and you did not edit your runtime settings before starting it, you will now need to change your settings. Click on the gear icon in the upper right to edit your Notebook Runtime. Set the values as specified below:
+    <table style="float:left">
+        <thead>
+            <tr><th>Option</th><th>Value</th></tr>
+        </thead>
+        <tbody>
+             <tr><td>ENVIRONMENT</td><td>Default (GATK 4.1.4.1, Python 3.7.7, R 4.0.3)</td></tr>
+              <tr><td>CPUs</td><td>4</td></tr>
+              <tr><td>Memory (GB)</td><td>15</td></tr>
+              <tr><td>Disk size (GB)</td><td>50</td></tr>
+              <tr><td>Startup script</td><td>(leave blank)</td></tr>
+              <tr><td>Compute Type</td><td>Standard VM</td></tr>
+        </tbody>
+    </table>
+    """
+# HTML tables should be followed by a new cell because otherwise text gets jammed next to them
+with herzog.Cell("markdown"):
+    """
+    Click the "Replace" button when you are done, and Terra will begin to create a new runtime with your settings. When it is finished, it will pop up asking you to apply the new settings.
+
+    ### Check kernel type
+
+    A kernel is a _computational engine_ that executes the code in the notebook. You can think of it as defining the programming language. For this notebook, we'll use a `Python 3` kernel. In the upper right corner of the notebook, just under the Notebook Runtime, it should say `Python 3`. If it doesn't, you can switch it by navigating to the Kernel menu and selecting `Change kernel`.
+
+    ### Install packages
+    """
+
+with herzog.Cell("python"):
+    #%pip install tenacity
+    pass
+
+with herzog.Cell("markdown"):
+    """
+    Make sure to restart the kernel whenever you pip install things in a Jupyter notebook.
+
+    ### Import all the packages this notebook will use
+    """
+
+with herzog.Cell("python"):
+    from firecloud import fiss
+    import tenacity
+    import pandas as pd
+    pd.set_option('display.max_row', 10)
+    import os
+    import io
+    import numpy as np
+    import seaborn as sns
+
+with herzog.Cell("markdown"):
+    """
+    ## Define filepaths and environmental variables
+    """
+
+with herzog.Cell("python"):
+    PROJECT = os.environ['GOOGLE_PROJECT']
+    PROJECT
+
+with herzog.Cell("python"):
+    WORKSPACE = os.path.basename(os.path.dirname(os.getcwd()))
+    WORKSPACE
+
+with herzog.Cell("python"):
+    bucket = os.environ['WORKSPACE_BUCKET']
+    bucket = bucket + '/'
+    bucket
+
+with herzog.Cell("markdown"):
+    """
+    Consolidate the Gen3 clinical entities into a single Terra data model using functions in the "terra_data_table_util" python notebook
```

Comment:
Consolidate the Gen3 clinical data tables into a consolidated data table using functions in the "terra_data_table_util" python notebook".

^trying to remove some jargon (Gen3 entities)